### PR TITLE
Add roles/permissions to section: Add Update Compliance to your Azure…

### PR DESCRIPTION
**Pull request by ricastil**
… subscription

adding the following information on roles required to enable, write or read in log analytics.

### CODE REVIEW

File: windows/deployment/update/update-compliance-get-started.md [hunk 1]
```diff
@@ -51,15 +51,21 @@ Before you begin the process to add Update Compliance to your Azure subscription
 
 ## Add Update Compliance to your Azure subscription
 
-Update Compliance is offered as an Azure Marketplace application which is linked to a new or existing [Azure Log Analytics](/azure/log-analytics/query-language/get-started-analytics-portal) workspace within your Azure subscription. To configure this, follow these steps:
+Update Compliance is offered as an Azure Marketplace application which is linked to a new or existing [Azure Log Analytics](/azure/log-analytics/query-language/get-started-analytics-portal) workspace within your Azure subscription. Please note that for the following steps you must have one of the following [Azure roles](https://docs.microsoft.com/en-us/azure/role-based-access-control/rbac-and-directory-admin-roles#azure-roles) at minimum in order to add the solution: Owner or Contributor.
```
Comment (JohanFreelancer9) [inline windows/deployment/update/update-compliance-get-started.md:54]:
```suggestion
Update Compliance is offered as an Azure Marketplace application that is linked to a new or existing [Azure Log Analytics](/azure/log-analytics/query-language/get-started-analytics-portal) workspace within your Azure subscription. Note that, for the following steps, you must have either an Owner or Contributor [Azure role](/azure/role-based-access-control/rbac-and-directory-admin-roles#azure-roles) as a minimum in order to add the solution.
```

File: windows/deployment/update/update-compliance-get-started.md [hunk 2]
```diff
@@ -51,15 +51,21 @@ Before you begin the process to add Update Compliance to your Azure subscription
...
 2. Select **Get it now**.
 3. Choose an existing or configure a new Log Analytics Workspace, ensuring it is in a **Compatible Log Analytics region** from the following table. Although an Azure subscription is required, you won't be charged for ingestion of Update Compliance data.
    - [Desktop Analytics](/sccm/desktop-analytics/overview) users should use the same workspace for Update Compliance.
    - [Azure Update Management](/azure/automation/automation-intro#update-management) users should use the same workspace for Update Compliance.
 4. After your workspace is configured and selected, select **Create**. You'll receive a notification when the solution has been successfully created.
 
+Once the solution is in place, a user can leverage one of the following Azure roles with Update Compliance:
+
+•	To edit and write queries we recommend the [Log Analytics Contributor ](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#log-analytics-contributor)
+•	To read and only view date we recommend the [Log Analytics Reader](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#log-analytics-reader)
```
Comment (JohanFreelancer9) [inline windows/deployment/update/update-compliance-get-started.md:67]:
```suggestion
Once the solution is in place, you can leverage one of the following Azure roles with Update Compliance:

- To edit and write queries we recommend the [Log Analytics Contributor](/azure/role-based-access-control/built-in-roles#log-analytics-contributor) role.

- To read and only view data we recommend the [Log Analytics Reader](/azure/role-based-access-control/built-in-roles#log-analytics-reader) role.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ricastil, kindly commit the suggested changes. Many thanks.

**Review by ricastil — COMMENTED** [review]
i accept the changes

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ricastil** [comment]
Hello @JohanFreelancer9 - When will we be able to see the changes in the public facing docs?

**Review by ricastil — COMMENTED** [review]
I accept the changes

**Comment by JohanFreelancer9** [comment]
@e0i, could you please assist to get this approved and merged? Thanks.

**Review by aczechowski — APPROVED** [review]
Thanks @ricastil for the contribution, and thanks @JohanFreelancer9 for the edits

**Pull request merged**